openff/toolkit/tests/test_molecule.py

```diff
@@ -2855,7 +2746,7 @@ def test_add_monovalent_lone_pair_virtual_site(self, molecule):
 
         # Try passing in a unitless distance
         with pytest.raises(AssertionError) as excinfo:
-            vsite1_index = molecule.add_monovalent_lone_pair_virtual_site(
```

Comment:
In lot's of these cases `vsite1_index` is unused causing the linter to complain. These removals are just linting.